src/server/api/index.js

```diff
@@ -223,23 +223,6 @@ function createAPI(app, mountPath, middlewareOpts) {
                 } else {
                     deferred.resolve(userData);
                 }
-            })
-            .catch(function (err) {
```

Comment:
I think a short '.catch(deferred.reject)' should remain here. (or does the nodeify take care of that?)

Reply:
Good catch